.github/workflows/go.yml

```diff
@@ -45,6 +45,8 @@ jobs:
           APP_KEY: ${{ secrets.APP_KEY }}
           AUTH_ID: ${{ secrets.AUTH_ID }}
           TEST_DATABASE: ${{ secrets.TEST_DATABASE }}
+          SECONDARY_ENGINE_CONNECTION_STRING: ${{ secrets.SECONDARY_ENGINE_CONNECTION_STRING }}
```

Comment:
Which engine is that?

Reply:
In actions? https://sdke2etestsecondary.eastus.kusto.windows.net

Reply:
Resolved offline